### ISSUE TITLE
Add Copy Script Tag with SRI option

### ISSUE THREAD

**Issue by kiwiant** [cdnjs/cdnjs#12746]
On the search results there is an option to **Copy Link** and **Copy Script Tag**. It would be great if there was another option so the Script Tag can be copied with the `integrity` and `crossorigin` attributes.

`<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>`

There are services to do this like srihash.org but its another step.

**Comment by kiwiant** [comment]
I see what's happening. From the home page search the libraries for say "JQuery" and you only have options to "Copy Url" and "Copy Script Tag". Click the library name to reveal all the versions and you also get the options to copy with the SRI.

It would be useful to include the SRI options on the search results page.

**Comment by dadon-david** [comment]
Hello everyone, 

I've just read [https://news.ycombinator.com/item?id=18559786](https://news.ycombinator.com/item?id=18559786)

TL;DR: Major sites has JavaScript tag without integrity attribute on their payment pages.

It would be very useful to have the SRI option on the library search page directly, instead of have to click on the library, then select the SRI option.

Dropdown menu on the search page (https://cdnjs.com/)[https://cdnjs.com/]
<img width="1293" alt="screenshot 2018-11-29 at 17 59 03" src="https://user-images.githubusercontent.com/8955104/49238900-bf835500-f401-11e8-9f98-7d288f1e4194.png">

Dropdown menu on the library page (https://cdnjs.com/libraries/jquery)[https://cdnjs.com/libraries/jquery]
<img width="1343" alt="screenshot 2018-11-29 at 17 59 23" src="https://user-images.githubusercontent.com/8955104/49238953-dc1f8d00-f401-11e8-8af7-cc906ed0e23a.png">


Thanks a lot,
